lib/language_pack/helpers/nodebin.rb

```diff
@@ -1,8 +1,8 @@
 require 'json'
 
 class LanguagePack::Helpers::Nodebin
-  NODE_VERSION = "20.9.0"
-  YARN_VERSION = "1.22.19"
+  NODE_VERSION = "22.110"
```

Comment:
```suggestion
  NODE_VERSION = "22.11.0"
```

Given that CI passed even with an invalid Node.js version here - it seems that there's missing test coverage?

Reply:
There is a test for this and it fails on this branch:

```
⛄️ 3.1.6 🚀 /Users/rschneeman/Documents/projects/work/heroku-buildpack-ruby (jwl/update-nodejs-default)
$ be rspec spec/helpers/node_installer_spec.rb
Run options: include {:focused=>true}

All examples were filtered out; ignoring {:focused=>true}
       Command: 'set -o pipefail; curl -L --fail --retry 5 --retry-delay 1 --connect-timeout 3 --max-time 30 https://nodejs.org/download/release/v22.110/node-v22.110-linux-arm64.tar.gz -s -o - | tar zxf - node-v22.110-linux-arm64/bin/node' failed on attempt 1 of 3.
       Command: 'set -o pipefail; curl -L --fail --retry 5 --retry-delay 1 --connect-timeout 3 --max-time 30 https://nodejs.org/download/release/v22.110/node-v22.110-linux-arm64.tar.gz -s -o - | tar zxf - node-v22.110-linux-arm64/bin/node' failed on attempt 2 of 3.
F       Command: 'set -o pipefail; curl -L --fail --retry 5 --retry-delay 1 --connect-timeout 3 --max-time 30 https://nodejs.org/download/release/v22.110/node-v22.110-linux-arm64.tar.gz -s -o - | tar zxf - node-v22.110-linux-arm64/bin/node' failed on attempt 1 of 3.
       Command: 'set -o pipefail; curl -L --fail --retry 5 --retry-delay 1 --connect-timeout 3 --max-time 30 https://nodejs.org/download/release/v22.110/node-v22.110-linux-arm64.tar.gz -s -o - | tar zxf - node-v22.110-linux-arm64/bin/node' failed on attempt 2 of 3.
F

Failures:

  1) LanguagePack::Helpers::NodeInstaller#install should extract a node binary on amd64
     Failure/Error: raise error_class, "Command: '#{command}' failed unexpectedly:\n#{result}"

     LanguagePack::Fetcher::FetchError:
       Command: 'set -o pipefail; curl -L --fail --retry 5 --retry-delay 1 --connect-timeout 3 --max-time 30 https://nodejs.org/download/release/v22.110/node-v22.110-linux-arm64.tar.gz -s -o - | tar zxf - node-v22.110-linux-arm64/bin/node' failed unexpectedly:
       tar: node-v22.110-linux-arm64/bin/node: Not found in archive
       tar: Error exit delayed from previous errors.
     # ./lib/language_pack/shell_helpers.rb:97:in `block in run!'
     # ./lib/language_pack/shell_helpers.rb:91:in `times'
     # ./lib/language_pack/shell_helpers.rb:91:in `run!'
     # ./lib/language_pack/fetcher.rb:31:in `fetch_untar'
     # ./lib/language_pack/helpers/node_installer.rb:24:in `block (2 levels) in install'
     # ./lib/language_pack/helpers/node_installer.rb:23:in `chdir'
     # ./lib/language_pack/helpers/node_installer.rb:23:in `block in install'
     # /Users/rschneeman/.rubies/ruby-3.1.6/lib/ruby/3.1.0/tmpdir.rb:96:in `mktmpdir'
     # ./lib/language_pack/helpers/node_installer.rb:20:in `install'
     # ./spec/helpers/node_installer_spec.rb:10:in `block (6 levels) in <top (required)>'
```

I've seen this before where the test fails when running the file (as above), but passes when running via `rspec-queue`. It could either be due to it not running the test at all or running it and not reporting it as a failure for some reason. I've not had the time to debug why.